lib/util/AbbreviationMappingsHelper.dart

```diff
@@ -0,0 +1,8 @@
+Map<String, String> position = {'RF' : 'Right Fielder', 'SS' : 'Short Stop', '1B' : 'First Baseman', '3B' : 'Third Baseman', 'CF' : 'Center Fielder', 'LF' : 'Left Fielder'};
+
+String getFullAthletePosition(String currentAthletePositon) {
```

Comment:
rename to something that shows that it only retrieves baseball positions

Reply:
Done